addons/survey/static/src/js/survey_choice_image_zoomer.js

```diff
@@ -0,0 +1,104 @@
+odoo.define('survey.choice_image_zoomer', function (require) {
+    'use strict';
+
+var publicWidget = require('web.public.widget');
+
+publicWidget.Widget.surveyChoiceImageZoomer = publicWidget.Widget.extend({
+    template: 'survey.survey_choice_image_zoomer',
+    xmlDependencies: ['/survey/static/src/xml/survey_choice_image_zoomer_templates.xml'],
+    events: {
+        'DOMMouseScroll .o_survey_img_zoomer_image': '_onImgScroll', // Firefox
+        'mousewheel .o_survey_img_zoomer_image': '_onImgScroll', // Chrome, Safari, IE
```

Comment:
Looks like "mousewheel" is deprecated in favor of "wheel":

https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event

Reply:
Good catch, I see that the other one is also only kept for retro compatibility. I think I can remove it :) https://developer.mozilla.org/en-US/docs/Web/API/Element/DOMMouseScroll_event